rules/klayout/drc/testing/run_sc_regression.py

```diff
@@ -189,7 +190,7 @@ def call_simulator(arg):
         os.remove("split_gds.rb")
 
     if os.path.exists("sc"):
-        os.system("rm -rf sc")
+        subprocess.run(["rm", "-rf", "sc"])
```

Comment:
This should use `shutils` and not call out to `rm`.

Reply:
@mithro Same as the above.